src/main/java/org/dcsa/ebl/repository/impl/ShippingInstructionCustomRepositoryImpl.java

```diff
@@ -39,77 +32,15 @@
     private final DatabaseClient client;
     private final R2dbcDialect r2dbcDialect;
```

Comment:
I don't think DatabaseClient and R2dbcDialect are used anymore and can be removed

Reply:
I am using them when I fetch the carrierBookingReferences as that part uses r2dbc directly